draft-ietf-tls-esni.md

```diff
@@ -631,15 +631,15 @@ structure available for the server, it SHOULD send a GREASE
   SHOULD vary to exercise all supported configurations, but MAY be held constant
   for successive connections to the same server in the same session.
 
-- Set the "enc" field to a randomly-generated valid encapsulated public key
-  output by the HPKE KEM.
-
 - Set the "record_digest" field to a randomly-generated string of hash_length
   bytes, where hash_length is the length of the hash function associated with
   the chosen cipher suite.
 
-- Set the "encrypted_client_hello" field to a randomly-generated string of
-  [TODO] bytes.
+- Set the "enc" field to a randomly-generated valid encapsulated public key
+  output by the HPKE KEM.
+
+- Set the "encrypted_ch" field to a randomly-generated string of padded_length
+  bytes.
```

Comment:
Is `padded_length` supposed to be the length for "encrypted_ch"? I was unclear on this.

Reply:
We should probably table this until we figure out the padding strategy. (Apparently that's still up in the air!) It also occurs to me that the first bullet in this list is obsolete -- we don't have a named group anymore or padded_length anymore:

```
      struct {
          CipherSuite suite;
          opaque record_digest<0..2^16-1>;
          opaque enc<1..2^16-1>;
          opaque encrypted_ch<1..2^16-1>;
      } ClientEncryptedCH;
```

Can we just say something like the following? 

"Set the "suite" field to a supported cipher suite. This selection SHOULD vary to exercise all supported configurations, but MAY be constant for successive connections to the same server in the same session."

Reply:
Ah, ok. I'll bring the [TODO] back for the length of encrypted_ch.